src/internet_identity/internet_identity.did

```diff
@@ -209,6 +209,31 @@ type RateLimitConfig = record {
     max_tokens: nat64;
 };
 
+// Captcha configuration
+type CaptchaConfig = record {
+    // Maximum number of unsolved captchas.
+    // Default: 500
```

Comment:
If there is a default, shouldn't it be optional?

Reply:
Ah, yes. Moved the default documentation to the entire `CaptchaConfig`.